src/entity/address.ts

```diff
@@ -0,0 +1,35 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { User } from './user';
+
+@Entity()
+export class Address {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  CEP: string;
+
+  @Column()
+  street: string;
+
+  @Column()
+  streetNumber: number;
+
+  @Column({ nullable: true })
+  complement: string;
```

Comment:
```suggestion
  @Column({ nullable: true })
  complement?: string;
```

É bom o TS ser consistente com o comportamento real.